cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/PythonAddDeclarationsPass.kt

```diff
@@ -200,6 +201,31 @@ class PythonAddDeclarationsPass(ctx: TranslationContext) : ComponentPass(ctx), L
                     assignExpression.declarations += handled
                 }
             }
+            (target as? InitializerListExpression)?.let {
+                it.initializers.forEach { initializer ->
+                    (initializer as? Reference)?.let {
+                        it.access = AccessValues.WRITE
+                        val handled = handleWriteToReference(initializer)
+                        if (handled is Declaration) {
+                            // We cannot assign an initializer here because this will lead to
```

Comment:
Duplicate code from above, can you please refactor this into a separate method? also the line breaks in the comments are off.